egs/chime4/s5_1ch/run.sh

```diff
@@ -6,26 +6,29 @@
 #                Inria (Emmanuel Vincent)
 #                Mitsubishi Electric Research Labs (Shinji Watanabe)
 #           2017 JHU CLSP (Szu-Jui Chen)
+#           2017 JHU CLSP (Aswin Shanmugam Subramanian)
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
 . ./path.sh
 . ./cmd.sh
+
 #####Baseline settings#####
 # Usage: 
-# 1. For using original baseline, execute './run.sh --baseline chime4_official'. 
-# We don't provide the function to train original baseline models anymore. Instead, we provided the
-# trained original baseline models in tools/ASR_models for directly using.
+# Execute './run.sh' to get the models.
+# We provide BLSTM masking based enhancement --enhancement single_BLSTMmask
 #
-# 2. For using advanced baseline, first execute './run.sh --baseline advanced --flatstart true' to
-# get the models. If you want to use DNN instead of TDNN, add option "--tdnn false".
-# Then execute './run.sh --baseline advanced' for your experiments. 
+# We stopped to support the old CHiME-3/4 baseline. If you want to reproduce the old results
+# Please use the old version of Kaldi, e.g., git checkout 9e8ff73648917836d0870c8f6fdd2ff4bdde384f
 
 # Config:
 stage=0 # resume training with --stage N
-
-baseline=advanced
-flatstart=false
-tdnn=true
+enhancement=single_BLSTMmask #### or your method
+# if the following options are true, they wouldn't train a model again and will only do decoding
+gmm_decode_only_flag=false
```

Comment:
please remove _flag to the end of these boolean options' names.  It's not our standard naming convention.